src/Middleware/Diagnostics.EntityFrameworkCore/test/FunctionalTests/Helpers/TestLoggerProvider.cs

```diff
@@ -29,14 +29,25 @@ public class TestLogger : ILogger
         {
             private List<string> _messages = new List<string>();
 
+            private object _sync = new object();
```

Comment:
```suggestion
private readonly object _sync = new();
```

So many 💯

Reply:
Agreed, but I don't think it's worth another PR to fix it as it's test only code.